projects/swimlane/ngx-ui/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Feature: Add `appearance` and `togglePosition` inputs to `SectionComponent`
 - Enhancement: When `sectionCollapsible` in `SectionComponent` is true, make the section header clickable to expand and collapse
 - Feature: Add `timeout` option to `ButtonComponent`
+- Fix(ngx-input): Make the lockpad button clickable when the textbox appearance is set to "Fill" and has `[unlockable] = "true"` set. Also fix vertical alignment.
```

Comment:
@diegocalme not sure if we want to include the scope in release notes. So instead of `Fix(ngx-input)` just `Fix`